hw/hw13/trigrams.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+import random
+
+open_file = open('sherlock.txt', 'r')
+text = open_file.read()
+
+open_file.close()
+
+#  remove unwanted characters
+for char in ["\\", '(', ')', '-', '.', ';']:
```

Comment:
You may want to add more to this list. Python has a string.punctuation constant you can use:
```
In [2]: import string

In [3]: string.punctuation
Out[3]: '!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~'
```
You can keep apostrophes and hyphens:
```
punc = string.punctuation
punc = string.punctuation.replace("'", "")  # keep apostropies
punc = string.punctuation.replace("-", "")  # keep hyphenated words
```
Your code as written leaves many quotation marks and other punctuation as words.